french/soiree/soiree.md

```diff
@@ -0,0 +1,3 @@
+Il y assez de pain d'�pice pour tous vos amis ! Invitez les tous pour une soir�e gourmande pain d'�pices. Avec du foie gras, �a sera encore mieux :)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.